<!-- -->
<launch>
	<arg name="FirstSignsListIds"			default=",9,10," />
	<arg name="FirstGreenTime"				default="15" />
	<arg name="SecondSignsListIds"			default=",7,8," />
	<arg name="SecondGreenTime"				default="15" />
	<arg name="FirstYellowTime"				default="5" />
	<arg name="SecondYellowTime"			default="5" />
		
	<node pkg="op_simulator_signs" type="op_simulator_signs" name="op_simulator_signs" output="screen">		
		<param name="FirstSignsListIds" 			value="$(arg FirstSignsListIds)" />		
		<param name="SecondSignsListIds" 			value="$(arg SecondSignsListIds)" />		
		<param name="FirstGreenTime" 				value="$(arg FirstGreenTime)" />
		<param name="SecondGreenTime" 				value="$(arg SecondGreenTime)" />
		<param name="FirstYellowTime" 				value="$(arg FirstYellowTime)" />
		<param name="SecondYellowTime" 				value="$(arg SecondYellowTime)" />
	</node>

</launch>
